src/lib.rs

```diff
@@ -16,6 +16,18 @@ pub struct Generator {
     pub(crate) counters: CounterSettings,
     pub(crate) fmt: bool,
     pub(crate) extra_op_enum_derives: Vec<syn::Path>,
+    pub(crate) error_server: GenerateErrorServer,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Default)]
+enum GenerateErrorServer {
+    /// Do not generate an error server under any circumstances
+    No,
+    /// Generate an error server if all functions have the same error type
+    #[default]
+    Maybe,
```

Comment:
This as default results in a bunch of warnings from hubris from things that are unused.

```
warning: associated function `new` is never used
   --> /hubris/target/thumbv7em-none-eabihf/release/build/task-dump-agent-9599de4fda05f5c7/out/server_stub.rs:552:12
    |
551 | impl FailServer {
    | --------------- associated function in this implementation
552 |     pub fn new(err: DumpAgentError) -> Self {
    |            ^^^
    |
    = note: `#[warn(dead_code)]` on by default

warning: `task-dump-agent` (bin "task-dump-agent") generated 1 warning
```

Should `No` be the default or `new` be marked `#[allow(dead_code)]`